packages/browser-wallet-api-helpers/README.md

```diff
@@ -56,16 +32,6 @@ detectConcordiumProvider()
     .catch(() => console.log('Connection to the Concordium browser wallet timed out.'));
 ```
 
-To include the injected types in the window object's type, one can include this package's `extend-window.d.ts` file when building.
-This can be acheived by adding the following to your project's `tsconfig.json` file:
-
-```json
-{
-    ...
-    "include": [..., "path/to/node_modules/@concordium/browser-wallet-api-types/extend-window.d.ts"]
-}
-```
-
```

Comment:
Is this section not needed anymore, or why has this been removed? It's stated that "the package changes the type of the window object", but it doesn't change the type until you include this file (or its equivalent in the `/lib` folder).

I realize we want to push using the detector to get the api, but people might still be interested in this anyway?

Reply:
Søren and I discussed it yesterday and the conclusion was that advanced users would figure this out themselves. For MetaMask people define their own global window in their application when accessing it from window directly. They do not include it like this. 

But I should update the documentation.

Reply:
I added a description for how to use `window.concordium` directly.